tests/e2e/specs/pacbio/pacbio_run_edit.cy.js

```diff
@@ -94,7 +94,7 @@ describe('Pacbio Run Edit view', () => {
     cy.wrap(PacbioRunFactory({ findBy: 'Revio' })).as('revioRunFactory')
     cy.get('@revioRunFactory').then((revioRunFactory) => {
       cy.intercept(
-        '/v1/pacbio/runs/1581?include=plates.wells.used_aliquots,plates.wells.libraries.tube,plates.wells.pools.tube,plates.wells.libraries.request,plates.wells.pools.requests,plates.wells.pools.libraries.request,plates.wells.pools.used_aliquots.tag,plates.wells.libraries.used_aliquots.tag,smrt_link_version',
+        '/v1/pacbio/runs/1581?include=plates.wells.used_aliquots,plates.wells.libraries.request,plates.wells.pools.requests,plates.wells.pools.libraries.request,plates.wells.pools.used_aliquots.tag,plates.wells.libraries.used_aliquots.tag,smrt_link_version',
```

Comment:
is there any scope for removing stuff from the service based on this work?

e.g. does library need to have a tube relationship now?

Reply:
The relationship in the service still needs to exist because a library has a barcode through the tube. The idea behind this work was to make barcode accessible directly in the pool/library API so we dont need to deal with tubes in the UI as we were only fetching them for the barcode.